sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorSender.java

```diff
@@ -125,9 +126,11 @@
             this.handler.getErrors().subscribe(error -> {
                 logger.error("[{}] Error occurred in sender error handler.", entityPath, error);
                 endpointStateSink.error(error);
-            }),
+            })
+        );
 
-            this.tokenManager.getAuthorizationResults().subscribe(
+        if (tokenManager != null) {
```

Comment:
This is for special Sender link called `coordinator` which create a transaction in SB and there is no auth for this according to service bus team because auth will be done when user want to use this transaction in messages. And user can not do anything with a transaction alone.